### PR TITLE
Update references of Parity -> OpenEthereum for tx service setup

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: docs/services_03_transactions.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ for running the project.
 
 Configure the parameters needed on <span style="color:#DB3A3D">`.env`</span>. These parameters **need to be changed**:
 - <span style="color:#DB3A3D">`ETHEREUM_NODE_URL`</span>: Http/s address of a ethereum node. It can be the same than <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>.
-- <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>: Http/s address of a Ethereum Parity node with
-[tracing enabled](https://wiki.parity.io/JSONRPC-trace-module).
+- <span style="color:#DB3A3D">`ETHEREUM_TRACING_NODE_URL`</span>: Http/s address of an Openethereum/Parity node with
```
Comment (tschubotz) [inline docs/services_03_transactions.md:28]: Small feedback, but they call it Open**E**thereum I think (https://github.com/openethereum/openethereum)

File: docs/services_03_transactions.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ curl 'http://localhost:8000/api/v1/about/'
...
 
-Run a Parity node in your local computer:
+Run a Openethereum node in your local computer:
 ```bash
-parity --chain goerli --tracing on --db-path=/media/ethereum/parity --unsafe-expose
+openethereum --chain goerli --tracing on --db-path=/media/ethereum/openethereum --unsafe-expose
 ```
 
-Edit <span style="color:#DB3A3D">`.env`</span> so docker points to he host Parity:
+Edit <span style="color:#DB3A3D">`.env`</span> so docker points to he host Openethereum:
```
Comment (tschubotz) [inline docs/services_03_transactions.md:56]: Open**E**thereum

**Review by tschubotz — APPROVED** [review]

File: docs/services_03_transactions.md (hunk 2 above)
Comment (tschubotz) [reply docs/services_03_transactions.md:56]: Oh and there is a typo. "to **t**he host :)

**Comment by Uxio0** [comment]
Thanks for the feedback @tschubotz !

**Pull request merged**